tests/vmi_configuration_test.go

```diff
@@ -1020,9 +1020,36 @@ var _ = Describe("Configurations", func() {
 			})
 		})
 
-		Context("[rfe_id:140][crit:medium][vendor:cnv-qe@redhat.com][level:component]with guestAgent", func() {
+		FContext("[rfe_id:140][crit:medium][vendor:cnv-qe@redhat.com][level:component]with guestAgent", func() {
 			var agentVMI *v1.VirtualMachineInstance
 
+			prepareAgentVM := func() *v1.VirtualMachineInstance {
+				// TODO: actually review this once the VM image is present
```

Comment:
What's up with this?

Reply:
This comment is not mine. My undrstanding is, that commented code will get deprecated once the VM image with GA installed is present.

Reply:
OK then. Although I'd love to rather see an issue referenced here.